lib/run.js

```diff
@@ -220,8 +220,10 @@ module.exports = function (options, callback) {
 
             // override the `done` event to fire the end callback
             callbacks.done = function (err) { // @todo - do some memory cleanup here?
+                var collectionDetails = _.pick(options.collection, ['id', 'name']);
+
                 emitter.emit('done', err, emitter.summary); // we now trigger actual done event which we had overridden
-                callback(err, emitter.summary);
+                callback(err, _.merge(emitter.summary, collectionDetails));
```

Comment:
Also, it might be good to add a check to ensure that these details are present in the summary, right around here:
https://github.com/postmanlabs/newman/pull/471/files#diff-de78b50723ff596d7bd35665f36a6cbfR166